packages/hooks-react/src/useSelectedLanguage.ts

```diff
@@ -0,0 +1,15 @@
+import { useTranslation } from 'react-i18next';
+import { useConfigStore } from '@jwp/ott-common/src/stores/ConfigStore';
+import { useMemo } from 'react';
+
+import env from '../../common/src/env';
+
+const useSelectedLanguage = (): string => {
```

Comment:
Hm, I am not sure we need to do it here as we already have this logic supported in `initI18n` function. So looks like we could simply use `i18n.language`.